src/notifications/types.ts

```diff
@@ -114,3 +114,10 @@ export type NotificationUserTopReaderContext = NotificationBaseContext & {
   userTopReader: Reference<UserTopReader>;
   keyword: Reference<Keyword>;
 };
+
+declare module 'fs' {
+  interface ReadStream {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    on(event: 'data', listener: (chunk: any) => void): this;
+  }
+}
```

Comment:
In node 22.x types they changed this so its always `Buffer` so you can't cast it, and since typeorm uses it for `query.stream()` I overwritten it here to `any` so we can type it as usual until Node supports generics here.